spec/model_patches_spec.rb

```diff
@@ -11,6 +11,46 @@
 
 end
 
+describe User, 'when patched by whatdotheyknow-theme' do
+
+  describe '#can_send_survey?' do
+    let(:user) { FactoryBot.create(:user) }
+    subject { user.can_send_survey? }
+
+    before do
+      allow(AlaveteliConfiguration).
+        to receive(:send_survey_mails).and_return(true)
+      allow(user).to receive(:survey).
+        and_return(double('survey', :already_done? => false))
+    end
+
+    it { is_expected.to eq(true) }
```

Comment:
Minor nitpick, but I'd make this its own context to make the "success" conditions more human readable:



```ruby
    #…
    subject { user.can_send_survey? }

    before do
      allow(AlaveteliConfiguration).
        to receive(:send_survey_mails).and_return(true)
    end

    context 'a survey has not been sent to an active user' do

      before do
        allow(user).to receive(:survey).
          and_return(double('survey', :already_done? => false))
      end

      it { is_expected.to eq(true) }
    end
```